WooCommerce/WooCommerceTests/Extensions/IconsTests.swift

```diff
@@ -505,6 +505,10 @@ final class IconsTests: XCTestCase {
         XCTAssertNotNil(UIImage.cloudImage)
     }
 
+    func testCopyBarButtonItemImageIconIsNotNil() {
```

Comment:
nit: maybe we use the snake case since it is the new testing method naming style, even though there are still a bunch of pre-existing test cases with the previous naming style? 🙂 

```suggestion
    func test_copy_bar_button_item_image_icon_is_not_nil() {
```

Reply:
Yes - this was updated in 945f0b7f5bd73941ef5ccebf59f83a1efa1086d9.